pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -62,6 +62,10 @@ func fixBundleCached() error {
 
 // Check if oc binary is cached or not
 func checkOcBinaryCached() error {
+	// Remove oc binary from older location and ignore the error
+	// We should remove this code after 3-4 releases.
```

Comment:
maybe add a date or a release version so that we have a starting point for the "3-4 releases"?

Reply:
@cfergeau If we are consist about releases then I can add the date but like this time we are not releasing so bit hard to put an date, I can add a tentative one if that helps?

Reply:
An approximate date corresponding to when the PR was written should be good enough
`// We should remove this code after 3-4 releases (after 2020-05-11)`